### ISSUE TITLE
fix(ux): add relationships to containers via bulk action even when source or target entity is absent

### ISSUE THREAD

**Issue by Archidoit** [LuatixHQ/opencti#9018]
## Description
A relationship is not added to a container if we add it via a bulk action and if one of the entities involved is not already present in the container.

## Reproducible Steps
- Go to Data > Relationship
- Select a relationship via the checkbox, and add it to a container via the 'add in container' toolbar button
![Image](https://github.com/user-attachments/assets/3badb3cc-653d-42d6-a27a-c4abadc6a925)

- Go to the container overview
- In the history we can see a line indicating the relationship has been added
![Image](https://github.com/user-attachments/assets/06c57aa1-7d5a-43a6-958a-e325c7c2582a)

- But in the Knowledge tab of the container, there is no relationship

The relationship is correctly added only if both the source entity and the target entity of the relationship are already present in the container.

**Comment by nino-filigran** [comment]
Not sure it's a real bug but anyway it's a behavior to fix: indeed, we do not have a place to only visualize realtions. Therefore, adding a relation in a container should automatically add the entities in the container.

**Comment by Archidoit** [comment]
@nino-filigran  Note that if both source and target entities of the relationship are in the container, the relationship seems to be added (see the Knowledge view)

**Comment by nino-filigran** [comment]
Indeed, I've tried this. To me, adding a relation without adding the from and to seems weird, hence the fix I've sugggested.

**Comment by SamuelHassine** [comment]
🤖 [AI-generated]

**Title updated:**
Old: `fix(ux): relationship not added in container via bulk action if source/target not present in container`
New: `fix(ux): add relationship to container via bulk action even when source or target entity is not yet present`

Description was a passive symptom statement — rewritten as an imperative verb phrase per the repo's contribution conventions.

**Comment by SamuelHassine** [comment]
🤖 [AI-generated]

Old title: `fix(ux): relationship not added in container via bulk action if source/target not present in container`
New title: `fix(ux): add relationship to container via bulk action even when source or target entity is absent`

Description started with a noun phrase ("relationship not added…") instead of an imperative verb — rewritten to lead with the imperative "add".

**Comment by SamuelHassine** [comment]
🤖 [AI-generated]

fix(ux): fix adding a relationship to a container via bulk action when source or target entity is absent → fix(ux): ensure relationships are added to containers via bulk action even when source or target entity is absent

Description started with "fix", redundantly echoing the type prefix; replaced with imperative "ensure" and reworded to active voice.

**Comment by SamuelHassine** [comment]
🤖 Convention fix: "ensure relationships are added to containers via bulk action even when source or target entity is absent" → "add relationships to containers via bulk action even when source or target entity is absent" — "ensure ... are added" is a passive embedded clause; replaced with direct imperative "add".
